pypeit/spectrographs/keck_lris.py

```diff
@@ -104,10 +105,67 @@ def header_keys(self):
             hdr_keys[0]['lampstat{:02d}'.format(kk+1)] = lamp_name
 
         return hdr_keys
+    '''
+
+
+    def init_meta(self):
+        """
+        Generate the meta data dict
+        Note that the children can add to this
+
+        Returns:
+            self.meta: dict (generated in place)
+
+        """
+        meta = {}
+        # Required (core)
+        meta['ra'] = dict(ext=0, card='RA')
+        meta['dec'] = dict(ext=0, card='DEC')
+        meta['target'] = dict(ext=0, card='TARGNAME')
+        meta['decker'] = dict(ext=0, card='SLITNAME')
+        meta['binning'] = dict(card=None, compound=True)
```

Comment:
Do we ever use the `compound` component of meta?  My rigid C brain is freaking out about some keys in these meta dictionaries not being defined in *all* dictionaries...

Reply:
Okay, found where it's used.  My C brain is still unhappy, but I'll deal...

Reply:
Fair point.  In hindsight, having a method that builds each of these
(and all the default keys) would be good practice.  e.g.

```
def mdict(ext=0, compound=False, etc...):
    new_dict = {}
    new_dict['ext'] = ext
    etc.
    return new_dict
```

I could be convinced to do this if so commanded.  ;)